cli_helper/kart.c

```diff
@@ -221,6 +232,10 @@ int main(int argc, char **argv, char **environ)
     {
         debug("enabled %s, pid=%d\n", cmd_path, getpid());
 
+        // Make this process the leader of a process group:
+        // The procress-group ID (pgid) will be the same as the pid.
+        setpgrp();
```

Comment:
If something is invoking kart (like a bash script/pipeline); will ctrl-c'ing it work? I'll do an experiment.